test/uk/gov/hmrc/entrydeclarationoutcome/controllers/OutcomeSubmissionControllerSpec.scala

```diff
@@ -52,15 +54,18 @@ class OutcomeSubmissionControllerSpec extends UnitSpec with MockOutcomeSubmissio
   "OutcomeSubmissionController postOutcome" should {
     "return CREATED" when {
       "request is handled successfully" in {
+        MockReportSender.timeFrom("E2E.total-e2eTimer", outcome.receivedDateTime)
```

Comment:
```suggestion
        val e2eDuration: Duration = Duration.ofSeconds(2)
        MockReportSender.timeFrom("E2E.total-e2eTimer", outcome.receivedDateTime) returns e2eDuration
```
and then use `Some(e2eDuration)` rather than `Some(null)` in the send report mocking below.